java/src/jmri/jmrit/display/layoutEditor/LayoutBlock.java

```diff
@@ -2860,7 +2860,7 @@ public int getNextBestBlock(Block previousBlock, Block destBlock, List<Integer>
                 if (!nextBlocks.contains(ro.getNextBlock())) {
                     // if(ro.getNextBlock()!=nextBlock){
                     int currentValue;
-                    if (routingMethod == LayoutBlockConnectivityTools.Metric.METRIC) {
+                    if (routingMethod == LayoutBlockConnectivityTools.Metric.HOPCOUNT) {
                         currentValue = routes.get(i).getMetric();
```

Comment:
Why this change? What does it do?
I'm not in any way against it, I'm only trying to understand it.